electron/src/main.ts

```diff
@@ -66,34 +70,34 @@ const BASE_URL = environment.web.getWebappUrl(argv.env);
 // Icon
 const ICON = `wire.${environment.platform.IS_WINDOWS ? 'ico' : 'png'}`;
 const ICON_PATH = path.join(APP_PATH, 'img', ICON);
-let tray = undefined;
+let tray: TrayHandler;
 
 let isFullScreen = false;
 let isQuitting = false;
-let main;
+let main: BrowserWindow;
 
 // IPC events
 const bindIpcEvents = () => {
-  ipcMain.on(EVENT_TYPE.ACTION.SAVE_PICTURE, (event, fileName, bytes) => {
-    download(fileName, bytes);
+  ipcMain.on(EVENT_TYPE.ACTION.SAVE_PICTURE, async (event: IpcMessageEvent, fileName: string, bytes: Uint8Array) => {
+    await download(fileName, bytes);
```

Comment:
Actual fix

Reply:
What's the fix? There is no subsequent call...

Reply:
As explained f2f, this is just to make TS happy.